rest/rest-server/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/WorkflowSubmitter.java

```diff
@@ -49,6 +53,9 @@
 
     private static final Logger logger = ProActiveLogger.getLogger(WorkflowSubmitter.class);
 
+    private static final String FILE_ENCODING = PASchedulerProperties.FILE_ENCODING.isSet() ? PASchedulerProperties.FILE_ENCODING.getValueAsString()
```

Comment:
Normally you don't need the call to isSet as this property has a default value. If it's not set in the config/scheduler/settings.ini it will simply use its default value "UTF-8"